skymp5-client/src/front/skympClient.ts

```diff
@@ -447,21 +446,35 @@ export class SkympClient {
     this.sendTarget.send({ t: MsgType.Host, remoteId }, false);
   }
 
-  private sendInputs() {
+  private updateData()
+  {
     const hosted =
       typeof storage["hosted"] === typeof [] ? storage["hosted"] : [];
     const targets = [undefined].concat(hosted as any);
-    //printConsole({ targets });
+    // removing all stop hosted objects from usedRefID array
+    deathSystem.usedRefID.filter((value) => targets.includes(value))
```

Comment:
This line does not work as you expect. Filter doesn't modify the original array.  https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/filter